src/halmos/sevm.py

```diff
@@ -625,7 +672,9 @@ def __init__(self, **kwargs) -> None:
         self.balance = kwargs["balance"]
         #
         self.block = kwargs["block"]
+        #
         self.context = kwargs["context"]
+        self.callback = kwargs["callback"]
```

Comment:
given that callback is optional (can be None), can we also make it optional in the kwargs?

something like `self.callback = kwargs.get("callback", None)`

Reply:
the callback is None only at the top-level.  i prefer to non-optional in the kwargs, so that i can catch earlier if callback is missing.